### PR TITLE
hardcode network_id with 0 value

### PR DESCRIPTION
**Pull request by boguslavsky**

### CODE REVIEW

File: src/blockchain/block.js [hunk 1]
```diff
@@ -56,12 +55,8 @@ export function verifyBlock (data, validators, networkId) {
     return false
   }
 
-  if (typeof networkId !== 'number' || networkId < 0 || networkId > 255) {
-    return false
-  }
-
   const Precommit = newMessage({
-    network_id: networkId,
+    network_id: NETWORK_ID,
```
Comment (slowli) [inline src/blockchain/block.js]: Shouldn't this field be removed?

File: src/types/message.js [hunk 2]
```diff
@@ -20,7 +22,7 @@ class NewMessage {
       }
     })
 
-    this.network_id = type.network_id
+    this.network_id = NETWORK_ID
```
Comment (slowli) [inline src/types/message.js]: I think you can use `NETWORK_ID` directly when creating `MessageHead`.

File: test/blockchain.js [hunk 3]
```diff
@@ -130,7 +124,6 @@ describe('Verify block of precommits', function () {
             'propose_hash': '1783d20a053b5c45b40e76358a51a7fce90eea391a409decfb9f9cbbb5a4875a',
             'block_hash': '1783d20a053b5c45b40e76358a51a7fce90eea391a409decfb9f9cbbb5a4875a'
           },
-          'network_id': 0,
```
Comment (slowli) [inline test/blockchain.js]: Off-topic: why are the object fields quoted here?

Comment (boguslavsky) [reply test/blockchain.js]: good catch, thanks

**Pull request merged**